.github/CODEOWNERS

```diff
@@ -171,6 +171,13 @@
 /code/modules/atmospherics/ @Pickle-Coding
 /code/modules/power/ @Pickle-Coding
 
+# ZephyrTFA
+
+/code/datums/json_savefile.dm @ZephyrTFA
+/code/datums/armor/ @ZephyrTFA
+/code/modules/client/preferences.dm @ZephyrTFA
```

Comment:
@Gamer025 Do you know if your system works with multiple people requesting codeowner over the same file? We have the "MULTIPLE OWNERS" section but I wish we didn't and want to know if we need to

Reply:
Okay well looks like I don't actually have codeowner on these because someone moved the files but didn't change the CODEOWNERS but let's pretend I did because I'm fixing it right after this

Reply:
Yep my action currently notifies all matched users, not just the last matching one like gitignore/codeowners is supposed to work
@stylemistake was thinking about reworking it at some point because this means its impossible to remove yourself by doing like
```
/code/ @UserNameHere
/code/DontWantThis.dm //No User name here so no notification
```
but that would require a major restructure of the codeowner file to be file grouped and not owner grouped
And even then its probably really easy to mess up and override someone else

Reply:
i'll take it i guess